Find a functionally equivalent code:android.util.ArrayMap.get:COMMENT
Method Modifier: public      
Comment:/**
 * Retrieve a value from the array.
 * @param key The key of the value to retrieve.
 * @return Returns the value associated with the given key,
 * or null if there is no such key.
 */

@@ -1,4 +1,4 @@
 {
-    final int index = key == null ? indexOfNull() : indexOf(key, key.hashCode());
+    final int index = indexOfKey(key);
     return index >= 0 ? (V) mArray[(index << 1) + 1] : null;
 }

